README.md

```diff
@@ -6,14 +6,14 @@ Using Harbor Compose is basically a four-step process.
 
 1. Define your app’s environment with a `Dockerfile` so it can be reproduced anywhere.
 
-2. Define the services that make up your app in `docker-compose.yml` so they can be run together in an isolated environment.  You can use the standard Docker Compose commands (like `docker-compose build` and `docker-compose up`) to build/run/test your Docker app locally.
+2. Define the services that make up your app in `docker-compose.yml` so they can be run together in an isolated environment.  You can use the standard docker compose commands (like `docker-compose build`, `docker-compose push`, `docker-compose up`, etc.) to build/run/test your Docker app locally.
```

Comment:
Why did you move away from using the proper noun for Docker Compose. In Docker's own Compose documentation, they use the proper noun when not referring to the actual commands. I think we should do the same. https://docs.docker.com/compose/overview/